lib/api/new.coffee

```diff
@@ -41,23 +41,23 @@ class New
     if not opts.path
       return W.reject(new Error('missing path'))
 
-    opts =
-      path: path.resolve(opts.path)
-      name: opts.template           ? global_config().get('default_template')
-      overrides: opts.overrides     ? {}
-      defaults: opts.defaults       ? { name: path.basename(opts.path) }
+    sprout = Sprout()
+    p = path.resolve(opts.path)
 
-    pkg = path.join(opts.path, 'package.json')
+    opts =
+      locals: opts.overrides ? {}
+      
+    pkg = path.join(p, 'package.json')
 
-    W.resolve(_.contains(sprout.list(), base_tpl_name))
+    W.resolve(_.contains(sprout.templates, base_tpl_name))
```

Comment:
@jenius this logic isn't working properly because `sprout.templates` output is different than what the old `sprout.list()` used to return